srcs/commands/Join.cpp

```diff
@@ -104,6 +107,11 @@ static int clientErrorChecks(Client *client, std::map<std::string, Channel*> &ch
 {
 	std::string hostname = client->getHostName();
 
+	if (channelName.empty())
```

Comment:
Is this something that should go to buffer or straight to client?

Reply:
It's an error message that they don't have it or I haven't found it yet. so it should go straight to client